extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -136,6 +138,13 @@ public Uni<TenantConfigContext> create(OidcTenantConfig config) {
                 });
     }
 
+    public void setupBackChannelLogout(RuntimeValue<Router> router) {
+        OidcConfig oidcConfig = Arc.container().instance(OidcConfig.class).get();
+        if (oidcConfig != null) {
```

Comment:
@michalvavrik There are some test cases where `Arc.container().instance(OidcConfig.class).get();` returns `null`
Is it correct to simply omit the `BackChannelLogoutHandler` if that happens or should, in this case, the `setupBackChannelLogout` not be called in the first place?
It returns `null` in e.g. `SecurityDisabledTestCase`, `CodeTenantReauthenticateTestCase`,...

Reply:
It is not, I don't understand how this can happen because you do all the good tings, but perhaps there is an easy way to fix it. Please look at the `io.quarkus.oidc.deployment.OidcBuildStep#addDefaultCacheBean` and do the same they do (consume this config in the build step and pass it down to the recorder).

Reply:
That seems to work fine actually.
E.g. in `SecurityDisabledTestCase` the config is now not null